grafana-plugin/src/models/organization/organization.ts

```diff
@@ -0,0 +1,30 @@
+import { action, observable } from 'mobx';
+
+import BaseStore from 'models/base_store';
+import { makeRequest } from 'network';
+import { RootStore } from 'state';
+
+import { Organization } from './organization.types';
+
+export class OrganizationStore extends BaseStore {
+  @observable
+  currentOrganization?: Organization;
+
+  constructor(rootStore: RootStore) {
+    super(rootStore);
+    this.path = '/organization/';
```

Comment:
this was changed from `/current_team/`, see [this change](https://github.com/grafana/oncall/pull/2611/files#diff-d87ee8df9c3c7eed2c4b729461f3831c5318b69907c58cdc0a2e566c170c2e9bR72-L72) on the backend